grc/templates/upload/_upload-files-section.html

```diff
@@ -32,7 +31,13 @@ <h2 class="govuk-heading-m">
 
                     <div class="govuk-summary-list__row">
                         <dd class="govuk-summary-list__value govuk-!-width-full" style="vertical-align: middle;">
+                            {% if evidence_file.password_required %}
+                            <img src="{{ url_for('static', filename='assets/images/padlock.svg') }}" style="width: 30px; vertical-align: bottom; margin-right: 2px;">
+                            {% endif %}

```

Comment:
There are two sections of code which are inconsistent.

This section shows the padlock symbol next to any password-protected files.
But in `upload/__init__.py`, we delete any password-protected files.
So this padlock will never be shown.

Can we have one (padlock symbol), or the other (auto-deleting password-protected files), but not both.

Let's talk to Tor and Spence/Alex to decide on the which workflow is clearest / most consistent.